### ISSUE TITLE
still error

### ISSUE THREAD

**Issue by hi-reeve** [wheatjs/vite-plugin-vue-type-imports#2]
after the first fix on #1 it does work on the vite config. but when i try to import a props type from different file i still got an error

![image](https://user-images.githubusercontent.com/45350572/140000553-bbb11458-e43e-43cd-9b75-a04ed3a6e663.png)

Component
```typescript
import { Props } from './Pagination';
const props = withDefaults(defineProps<Props>(), {
    currentPage: 1,
    perPage: 10,
    paginationSize: 5,
    rangeSize: 1,
    hideSkipButton: false,
});
```

Pagination.ts
```typescript
export interface Props {
    currentPage: number;
    paginationSize?: number;
    perPage?: number;
    rangeSize?: number;
    total: number;
    hideSkipButton?: boolean;
}
```

**Comment by wheatjs** [comment]
😬 Sorry about that. Could you provide a minimal reproduction? I haven't been able to reproduce this myself

**Comment by hi-reeve** [comment]
hey! sorry for the late reply, been busy with stuff
here is my branches repo.

https://github.com/zynth17/vitailse/tree/bug-vite-plugin-vue-type-import

**Comment by wheatjs** [comment]
Are you getting that error in your repo? I pulled it and ran it with the `<HelloWorld />` component and didn't run into any errors? Can I ask what platform you are on?

**Comment by hi-reeve** [comment]
> Are you getting that error in your repo? I pulled it and ran it with the `<HelloWorld />` component and didn't run into any errors? Can I ask what platform you are on?

yes in that repo im getting the error as well
basically it just doesn't work in my machine.
i'm currently on windows 11.

**Comment by hi-reeve** [comment]
hello, any update on this?

**Comment by wheatjs** [comment]
Sorry for the delay, I've been pretty busy recently. I'll try to get a fix out this weekend!

**Comment by hi-reeve** [comment]
thankyou so much!

**Comment by DannyFeliz** [comment]
Just tried and I'm getting this error too

**Comment by wheatjs** [comment]
Sorry it took so long, should be fixed in [v0.0.3 ](https://github.com/wheatjs/vite-plugin-vue-type-imports/releases/tag/v0.0.3)

**Comment by hi-reeve** [comment]
![image](https://user-images.githubusercontent.com/45350572/143528661-fb536af1-edf4-4df5-9b70-a958815397fa.png)

still got the same error even after upgrading to v0.0.3.
can you re open this?

**Comment by wheatjs** [comment]
Can you provide a reproduction for that? It works in the original reproduction you provided

**Comment by hi-reeve** [comment]
> Can you provide a reproduction for that? It works in the original reproduction you provided

weirdly it does work on my reproduction i provided. here's the reproduction on stackblitz
https://stackblitz.com/edit/vitejs-vite-c5b9xa?file=src/components/HelloWorld.vue

**Comment by wheatjs** [comment]
Take a look at this https://stackblitz.com/edit/vitejs-vite-xz3znc?file=src%2Fcomponents%2FHelloWorld.vue

I forked your stackblitz and installed the plugin in the vite.config.ts file and it works. Maybe you forgot to do that?

**Comment by hi-reeve** [comment]
> Take a look at this https://stackblitz.com/edit/vitejs-vite-xz3znc?file=src%2Fcomponents%2FHelloWorld.vue
> 
> 
> 
> I forked your stackblitz and installed the plugin in the vite.config.ts file and it works. Maybe you forgot to do that?

oh sorry i forgot to do it on the reproduction. 
and no i did not forget it on my project. i will try it once again and will tell you afterwards. thankyou.

**Comment by wheatjs** [comment]
@zynth17 I just did a big update to the plugin, maybe you could update to the latest version and see if it works for you now

**Comment by hi-reeve** [comment]
> @zynth17 I just did a big update to the plugin, maybe you could update to the latest version and see if it works for you now

the new version works! thankyou so much!

**Comment by cizkey** [comment]
https://stackblitz.com/edit/vitejs-vite-xz3znc?file=src%2Fcomponents%2FHelloWorld.vue

@zynth17 i try to run your demo with '0.1.2', but it still error. i  only modify some lib version. 
![image](https://user-images.githubusercontent.com/14026236/145672653-1e6c5c29-c571-4232-905d-b9ae6b8f3225.png)


while when i changed the versiono '0.0.3' , the demo run work fun! 
![image](https://user-images.githubusercontent.com/14026236/145672937-66d5158b-8211-40ed-8b32-b82c95d583a4.png)

@wheatjs can you help me?

**Comment by wheatjs** [comment]
It looks like it is a problem with adding an extension to the import. Replace `import { Props } from '../types/HelloWorld.ts'` with `import { Props } from '../types/HelloWorld'`.

I'll see if I can get a fix out for this tomorrow, thanks!

**Comment by wheatjs** [comment]
@cizkey solution above in case you didn't get the notification since this issue is closed 🤔

**Comment by cizkey** [comment]
> @cizkey solution above in case you didn't get the notification since this issue is closed 🤔

thanks , i knew it.

**Comment by wheatjs** [comment]
@cizkey Should be fixed in version 0.1.3
